### PR TITLE
Bugfix/remove linter erros

### PR DESCRIPTION
**Pull request by greatSumini**
## Description

remove linter errors, update snapshots for Badge, DatePicker component

## Related Issues

x

## Tests

I added the following tests:

x

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: main/Badge/index.tsx [hunk 1]
```diff
@@ -22,38 +24,44 @@ export interface BadgeProps extends StyleProps{
 const StyledView = styled.View<StyleProps>`
   position: absolute;
   top: -15px;
-  ${(props) => props.badgePlacement}: -10px;
+
```
Comment (JeffGuKang) [inline main/Badge/index.tsx]: Meaningless empty line.

File: main/Badge/index.tsx [hunk 2]
```diff
@@ -1,7 +1,9 @@
 import React, { FC } from 'react';
```
Comment (JeffGuKang) [inline main/Badge/index.tsx]: Remove empty line.

**Review by JeffGuKang — CHANGES_REQUESTED** [review]
`DatePicker` is waiting the PR related to #337. So how about dealing with only Badges here?

**Comment by greatSumini** [comment]
> `DatePicker` is waiting the PR related to #337. So how about dealing with only Badges here?

Sounds great! i reverted commits about DatePicker

File: main/Badge/index.tsx (hunk 1 above)
Comment (greatSumini) [reply main/Badge/index.tsx]: thk u for comment! resolved in 4330880

File: main/Badge/index.tsx (hunk 2 above)
Comment (greatSumini) [reply main/Badge/index.tsx]: resolved in 4330880

**Comment by JeffGuKang** [comment]
@greatSumini Good! Could you update a doc? Cause `position` should be replaced by `badgePlacement`.

**Comment by greatSumini** [comment]
> @greatSumini Good! Could you update a doc? Cause `position` should be replaced by `badgePlacement`.

resolved in 62b32b7 😄

**Review by JeffGuKang — APPROVED** [review]

**Pull request merged**